### ISSUE TITLE
error en paraules agudes acabades en -iu

### ISSUE THREAD

**Issue by jaumeortola** [Softcatala/separador-sillabes#11]
No les marquem com a agudes.

Exemples: 
cicatriu, seguiu, doiut, Guiu, kikuiu, enfranquiu, esquiu, esquius, reguiu, regadiu...

Lloc rellevant del codi: https://github.com/Softcatala/separador-sillabes/blob/b6d270bb89db63a2111aa638f3b30aff987305af/hyphen-softcatala.js#L228

**Comment by jmontane** [comment]
Sembla que també passa amb les acabades en -ui

Exemple: avui

Sembla que cal considerar 2 casos addicionals de paraula aguda amb diftong:


    else if (syll[ns - 1].match(/[aeo][iu][^aeiou]*$/i)) {
        r = 0;
    } // aguda (amb diftong)
    else if (syll[ns - 1].match(/iu[^aeiou]*$/i)) {
        alert(syll[ns-1])
        r = 0;
    } // aguda (amb diftong): cicatriu
    else if (syll[ns - 1].match(/[^gq]ui[^aeiou]*$/i)) {
        alert(syll[ns-1])
        r = 0;
    } // aguda (amb diftong): avui

**Comment by jaumeortola** [comment]
Sí, gràcies, una cosa així. Hauríem de fer unes quantes proves, i potser afegir un grapat de tests al codi.
